CMakePresets.json

```diff
@@ -0,0 +1,125 @@
+        "CMAKE_CXX_FLAGS_DEBUG": {
+          "type": "STRING",
+          "value": "/MDd /RTCsu /Od /Ob0 /GS /D_DEBUG /DNBS_ENABLE_MEMORY_LEAK_DETECTION"
+        },
+        "CMAKE_CXX_FLAGS_MINSIZEREL": {
+          "type": "STRING",
+          "value": "/MD /O1 /Os /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"
+        },
+        "CMAKE_CXX_FLAGS_RELEASE": {
+          "type": "STRING",
+          "value": "/MD /O2 /Oi /Ot /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"
+        },
+        "CMAKE_CXX_FLAGS_RELWITHDEBINFO": {
+          "type": "STRING",
+          "value": "/MD /O2 /Oi /Ot /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"
+        },
+        "CMAKE_CXX_FLAGS_USABLEDEBUG": {
+          "type": "STRING",
+          "value": "/MD /O1 /Od /Ob1 /Gs /DNDEBUG /DQT_FORCE_ASSERTS"
+        },
+        "CMAKE_CXX_STANDARD_LIBRARIES": {
+          "type": "STRING",
+          "value": "kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib"
+        },
+        "CMAKE_C_FLAGS": {
+          "type": "STRING",
+          "value": "/Zi /MP /W3 /GS /Gy /Zc:wchar_t /Gm- /fp:precise /errorReport:none /GF /WX- /Zc:forScope /Gd /EHa /nologo /DNOMINMAX /DNO_WARN_MBCS_MFC_DEPRECATION /D_CRT_SECURE_NO_DEPRECATE /D_SCL_SECURE_NO_WARNINGS /DWIN32 /D_MBCS /DWINVER=0x0A00"
+        },
+        "CMAKE_C_FLAGS_DEBUG": {
+          "type": "STRING",
+          "value": "/MDd /RTCsu /Od /Ob0 /D_DEBUG /DNBS_ENABLE_MEMORY_LEAK_DETECTION"
+        },
+        "CMAKE_C_FLAGS_MINSIZEREL": {
+          "type": "STRING",
+          "value": "/MD /O1 /Os /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"
+        },
+        "CMAKE_C_FLAGS_RELEASE": {
+          "type": "STRING",
+          "value": "/MD /O2 /Oi /Ot /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"
+        },
+        "CMAKE_C_FLAGS_RELWITHDEBINFO": {
+          "type": "STRING",
+          "value": "/MD /O2 /Oi /Ot /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"
+        },
+        "CMAKE_C_FLAGS_USABLEDEBUG": {
+          "type": "STRING",
+          "value": "/MD /O1 /Od /Ob1 /Gs  /DNDEBUG"
+        },
+        "CMAKE_C_STANDARD_LIBRARIES": {
+          "type": "STRING",
+          "value": "kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib"
+        },
+        "CMAKE_INSTALL_LIBDIR": {
+          "type": "PATH",
+          "value": "lib/${dollar}{CMAKE_INSTALL_CONFIG_NAME}"
+        },
+        "CMAKE_INSTALL_PREFIX": {
+          "type": "PATH",
+          "value": "install"
+        },
+        "CMAKE_STATIC_LINKER_FLAGS": {
+          "type": "STRING",
+          "value": "/machine:x64 /NOLOGO /NODEFAULTLIB:libcmt.lib"
+        },
+        "CMAKE_STATIC_LINKER_FLAGS_DEBUG": {
+          "type": "STRING",
+          "value": ""
+        },
+        "CMAKE_STATIC_LINKER_FLAGS_USABLEDEBUG": {
+          "type": "STRING",
+          "value": ""
+        },
+        "CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL": {
+          "type": "STRING",
+          "value": "/LTCG"

```

Comment:
Remove all /LTCG (Link-time code generation) flags. Flag prevents using library in different versions of the visual studio (see "ITK Build HowTo.txt" in https://svn.nexstim.local/svn/ExternalLibs/ITK/ITK-5_0_1/MSVC142_X64 and https://jira.nexstim.local:8443/browse/SNBT-233).